src/components/screens/CollectionsView.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <DefaultLayout>
+    <div>
+      <div class="mave-1000px-col">
+        <div class="mave-screen-title-bar">
+          <div class="mave-screen-title">My saved collections</div>
+        </div>
+        <div v-if="collectionsStatus == 'Loaded' && collections.length > 0">
+          <DataTable
+            class="mave-collections-table"
+            data-key="urn"
+            :multi-sort-meta="[{field: 'role', order: 1}, {field: 'name', order: 1}]"
+            sort-mode="multiple"
+            :value="collections"
+          >
+            <Column field="name" header="Collection name" :sortable="true">
+              <template #body="{data}"><router-link :to="{name: 'collection', params: {urn: data.urn}}">{{data.name}}</router-link></template>
+            </Column>
+            <Column class="mave-collection-description" field="description" header="Description" :sortable="true" />
+            <Column body-class="mave-align-center" :field="(c) => (c.experimentUrns || []).length" header="Experiments" :sortable="true" />
+            <Column body-class="mave-align-center" :field="(c) => (c.scoreSetUrns || []).length" header="Score&nbsp;sets" :sortable="true" />
+            <Column field="role" header="Permissions" :sortable="true" />
+            <Column field="creationDate" header="Created" :sortable="true">
+              <template #body="{data}">{{ formatDate(data.creationDate) }}</template>
+            </Column>
```

Comment:
Perhaps a last updated field would be useful as well? It seems like it would be more often useful than a created date.